vault/init.go

```diff
@@ -461,9 +462,11 @@ func (c *Core) UnsealWithStoredKeys(ctx context.Context) error {
 	// This usually happens when auto-unseal is configured, but the servers have
 	// not been initialized yet.
 	if len(keys) == 0 {
```

Comment:
Are we actually calling into UnsealWithStoredKeys from diagnose?  I think we want to avoid actually unsealing.

Reply:
I don't think we are. Is there somewhere you suspect this might be getting called?